src/data/complex/basic.lean

```diff
@@ -117,6 +133,7 @@ def conj (z : ℂ) : ℂ := ⟨z.re, -z.im⟩
 @[simp] lemma conj_zero : conj 0 = 0 := ext_iff.2 $ by simp [conj]
 @[simp] lemma conj_one : conj 1 = 1 := ext_iff.2 $ by simp
 @[simp] lemma conj_I : conj I = -I := ext_iff.2 $ by simp
+@[simp] lemma conj_two : conj 2 = 2 := ext_iff.2 $ by simp
```

Comment:
Why this lemma? Shouldn't we have `conj_bit0` and `conj_bit1`?

Reply:
I was puzzled too. It's only used in [data.complex.exponential](https://github.com/leanprover-community/mathlib/blob/f78693db5f4d14cf6335ac304c8eb5542330d18d/src/data/complex/exponential.lean#L549) when proving `sinh (conj x) = conj (sinh x)` (and the analogous statement for `cosh`).

Reply:
Let's try adding `conj_bit0` and `conj_bit1` and hoping we can then drop `conj_two`.

Reply:
Done!